src/analysis/retail/evoker/preservation/modules/talents/Stasis.tsx

```diff
@@ -624,21 +612,14 @@ class Stasis extends Analyzer {
         {this.selectedCombatant.hasTalent(TALENTS_EVOKER.ENGULF_TALENT) && (
           <div>
             <br />
-            As Flameshaper, the only viable <SpellLink spell={TALENTS_EVOKER.STASIS_TALENT} /> usage
-            is to store
-            <ol>
-              <li>
-                <SpellLink spell={TALENTS_EVOKER.DREAM_BREATH_TALENT} />
-              </li>
-              <li>
-                <SpellLink spell={TALENTS_EVOKER.ENGULF_TALENT} />
-              </li>
-              <li>
-                <SpellLink spell={TALENTS_EVOKER.ENGULF_TALENT} />
-              </li>
-            </ol>
-            Try to make sure to use <SpellLink spell={TALENTS_EVOKER.ENGULF_TALENT} /> on a target
-            that you can guarantee will have{' '}
+            As Flameshaper, your <SpellLink spell={TALENTS_EVOKER.STASIS_TALENT} /> should always
+            contain two casts of <SpellLink spell={TALENTS_EVOKER.ENGULF_TALENT} />, the final spell
+            can be either a <SpellLink spell={TALENTS_EVOKER.DREAM_BREATH_TALENT} /> in the first
+            slot, or a <SpellLink spell={TALENTS_EVOKER.TEMPORAL_ANOMALY_TALENT} /> in the final
+            one.
```

Comment:
This phrasing is a little bit confusing imo, might just be easier to make it more concrete and say something like

```
As Flameshaper, there are 2 valid <SpellLink spell={TALENTS_EVOKER.STASIS_TALENT}/> sequences, either <SpellLink spell={TALENTS_EVOKER.ENGULF_TALENT}/> -> <SpellLink spell={TALENTS_EVOKER.ENGULF_TALENT}/> -> <SpellLink spell={TALENTS_EVOKER.TEMPORAL_ANOMALY_TALENT}/> or <SpellLink spell={TALENTS_EVOKER.DREAM_BREACH_TALENT}/> -> <SpellLink spell={TALENTS_EVOKER.ENGULF_TALENT}/> -> <SpellLink spell={TALENTS_EVOKER.ENGULF_TALENT}/>
```

or some variation of it with bullet points or just `SpellIcon` to be concise